_commands/toys/t.md

```diff
@@ -0,0 +1,219 @@
+---
+---
+
+t
+-------
+
+`t` lets you interface with Twitter from the command line.
+
+~~~ bash
+$ t update "tweet tweet"
+$ t stream timeline -l
+~~~
+
+<!--more-->
+
+t is a command line utility (CLI) for interacting with Twitter.
+t is not typically bundled with an OS and thus requires installation. In order to install t you must already have Ruby installed. t requires a Twitter API token for each authorized user account.
+
+t capitalises on the ability to string UNIX commands, vastly increasing its usefulness. (xargs may be required)
+
+Full documentation on t is available on the project's GitHub page: <a href="https://github.com/sferik/t">https://github.com/sferik/t</a>
```

Comment:
Doesn't Markdown have a more succinct linking syntax? You should use that instead.